### DISCUSSION TITLE
There is a problem that the Role of Google SAML account in Kimai is initialized when logged in again after logging out.

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4757] [Plugins]
![스크린샷 2024-04-09 113619](https://github.com/kimai/kimai/assets/165639563/c7cfd494-7504-4107-9bc1-1898f5ccfde3)

After you set up Role (Teamlead, Administrator, System-Admin) on another Google SAML account with a System-Admin account,
If you log in with a Google SAML account that has been granted Role, and then log in again, there is a problem that Role is initialized.
What should I do with this problem?

Help me!

**Comment by kevinpapst** [comment]
I don't understand the problem you describe.
Can you rephrase that? 
I love Deepl, it helps me write better copy, maybe give it a try: https://www.deepl.com/write

**Comment by kevinpapst** [comment]
Ah, after reading your other issue, I think I understand the problem. Change this setting:
<img width="497" alt="Bildschirmfoto 2024-04-09 um 15 39 18" src="https://github.com/kimai/kimai/assets/533162/1e3af6ab-4b78-49ff-baec-996082464954">

**Reply by seleondevel** [reply]
@kevinpapst Oh! Thank you for your answer! I solved it!
